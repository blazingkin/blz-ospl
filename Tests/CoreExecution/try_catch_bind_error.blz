:main
    try
        asdf()
        print("No error")
    catch
        print(error)
    end

    try
        this_throws_an_error()
    catch
        print(error)
    end

end

:this_throws_an_error
    throw "An error message"
end