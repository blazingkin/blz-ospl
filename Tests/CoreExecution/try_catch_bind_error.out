Error occurred on line 3
Could not find a value for asdf
Did you mean "abs"?
An error message
